packages/web/components/Patron/PatronTile.tsx

```diff
@@ -60,11 +57,26 @@ export const PatronTile: React.FC<Props> = ({ index, patron }) => {
       },
     [player.profile?.timeZone],
   );
-  const description = getPlayerDescription(player);
-  const displayDescription =
-    description && description.length > MAX_BIO_LENGTH
+
+  const displayDescription = useMemo(() => {
+    const description = getPlayerDescription(player);
+    return description && description.length > MAX_BIO_LENGTH
```

Comment:
I know it's outside the purview of this PR, but I really think we should be rendering the description as Markdown like it is on the profile details page and limiting its length using [`line-clamp`](https://www.w3docs.com/learn-css/line-clamp.html) which works like `text-overflow: ellipsis`, but for multiple lines.

If someone has paragraph breaks in their description, smooshing it all together can make it uglier.